textfield/styles/index.styl

```diff
@@ -19,12 +23,9 @@
   :invalid
     box-shadow                              none
 
-  textarea
-    vertical-align                          top
-
-  &.disabled
+  &--disabled
     .mdl-textfield__label:after
       display                               none
     
     .mdl-textfield__input
-      color                                 mdl-light-full
+      color                                 rgba(0,0,0,0.26)
```

Comment:
The reasoning was that mdl-light-full is a variable from another project so this didn't really do what it tried to do. I imagine it would be good to have a config for all components instead of hardcoding, maybe we should have an issue for that?